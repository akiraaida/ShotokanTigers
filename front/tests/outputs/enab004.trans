10 admin                00000 00000.00 A 
09 Bran Bob             12347 00000.00   
01 Bran Bob             12347 00005.00   
00                      00000 00000.00   
10 Bran Bob             00000 00000.00 S 
01 Bran Bob             12347 00005.00   
01 Bran Bob             12347 00000.10   
00                      00000 00000.00   
