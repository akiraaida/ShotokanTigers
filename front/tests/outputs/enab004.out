Please enter a command: 
Please enter your session type: 
Please enter a command: 
Please enter the account holder's name: 
Please enter the user's account number: 
The enable transaction has completed.
Please enter a command: 
Please enter the account holder's name: 
Please enter the user's account number: 
Please enter amount to withdraw: 
The withdrawal transaction has completed.
Available Balance (12347):
105.00
Please enter a command: 
Please enter a command: 
Please enter your session type: 
Please enter a login name: 
Please enter a command: 
Please enter the user's account number: 
Please enter amount to withdraw: 
The withdrawal transaction has completed.
Available Balance (12347):
99.90
Daily Withdrawals Remaining (12347):
495.00
Please enter a command: 
